docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastForest.tt

```diff
@@ -1,12 +1,17 @@
 ﻿<#@ include file="RegressionSamplesTemplate.ttinclude"#>
 
 <#+
-string ClassHeader = @"// This example requires installation of additional NuGet package
-        // <a href=""https://www.nuget.org/packages/Microsoft.ML.FastTree/"">Microsoft.ML.FastTree</a>. ";
+string ClassHeader = @"
+        // This example requires installation of additional NuGet package for
```

Comment:
Since you put this on a new line, it added an extra empty line for all the .cs files that use this .tt file. If you fix this and run custom tool, it'll fix the other cases too (I commented about one)